managed/CounterStrikeSharp.API/Modules/Commands/Targeting/Target.cs

```diff
@@ -83,11 +83,11 @@ public Target(string target)
         }
     }
 
-    private bool TargetPredicate(CCSPlayerController player, CCSPlayerController? caller, CCSGameRules gameRules)
+    private bool TargetPredicate(CCSPlayerController player, CCSPlayerController? caller, CCSGameRules? gameRules)
```

Comment:
`CCSGameRules? gameRules = null`

Reply:
We send null gamerules already. Do we need it?